### PR TITLE
CVSB-756

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import { IonicStorageModule } from "@ionic/storage";
 import { StorageService } from "../providers/natives/storage.service";
 import { AuthService } from "../providers/global/auth.service";
 import { OpenNativeSettings } from "@ionic-native/open-native-settings";
-import { VehicleTestService } from "../providers/vehicle-test.service";
+import { TestTypeOldService } from "../providers/test-type-old.service";
```
Comment (danieliordache) [inline src/app/app.module.ts]: remove ...Old... from Service Name please

File: src/pages/testing/test-creation/vehicle/vehicle-history/vehicle-history.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!--
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-history/vehicle-history.html]: remove comments

File: src/pages/testing/test-creation/vehicle/vehicle-history/vehicle-history.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+
+/**
+ * Generated class for the VehicleHistoryPage page.
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-history/vehicle-history.ts]: remove comments

File: src/providers/global/http.service.ts [hunk 4]
```diff
@@ -28,7 +30,8 @@ export class HTTPService {
   }
 
   getTestTypes(): Observable<TestTypesModel[]> {
-    return this.http.get<TestTypesModel[]>(AppConfig.BACKEND_URL_TESTTYPES);
+    // return this.http.get<TestTypesModel[]>(AppConfig.BACKEND_URL_TESTTYPES);
```
Comment (danieliordache) [inline src/providers/global/http.service.ts]: use the data from the microservice

Comment (vladddragomir) [reply src/providers/global/http.service.ts]: done

File: src/pages/testing/test-creation/vehicle/vehicle-history/vehicle-history.ts (hunk 3 above)
Comment (vladddragomir) [reply src/pages/testing/test-creation/vehicle/vehicle-history/vehicle-history.ts]: done

File: src/pages/testing/test-creation/vehicle/vehicle-history/vehicle-history.html (hunk 2 above)
Comment (vladddragomir) [reply src/pages/testing/test-creation/vehicle/vehicle-history/vehicle-history.html]: done

File: src/app/app.module.ts (hunk 1 above)
Comment (vladddragomir) [reply src/app/app.module.ts]: done

**Pull request merged**